components/kyma-environment-broker/internal/process/input/input.go

```diff
@@ -233,6 +233,7 @@ func (r *RuntimeInput) applyProvisioningParametersForProvisionRuntime() error {
 	if params.LicenceType != nil {
 		r.provisionRuntimeInput.ClusterConfig.GardenerConfig.LicenceType = params.LicenceType
 	}
+	r.provisionRuntimeInput.ClusterConfig.Administrators = []*string{&r.provisioningParameters.ErsContext.UserID}
```

Comment:
please add short test to `input_test.go`

Reply:
Done